sources/packages/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -140,6 +140,15 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
           );
         }),
       )
+      .orderBy(
+        `CASE 
+          WHEN EXISTS(${this.studentAppealRequestsService
+            .appealsByStatusQueryObject(StudentAppealStatus.Pending)
+            .getSql()}) THEN 1
+              ELSE 2
+            END`,
+      )
+      .addOrderBy("studentAppeal.submittedDate", "DESC")
```

Comment:
Do we still need to use the getRawMany() because it is not a union anymore?

Reply:
we were not using union. `getRawMany()` is used because of subqueries

Reply:
We can use the getRawAndEntities also now as an alternative with the helper to populate the object model. I would say that this would be the preferred way.